office/libreoffice/Portfile

```diff
@@ -41,18 +41,18 @@ distfiles           ${main_distfile}:main \
                     dtoa-20180411.tgz:dtoa \
                     f543e6e2d7275557a839a164941c0a86e5f2c3f2a0042bfc434c88c6dde9e140-opens___.ttf:opens
 
-checksums           libreoffice-7.1.1.1.tar.xz \
-                    rmd160  15cf715de67329f4e4956763d836ee927f513357 \
-                    sha256  5f5fcf6fc492ec40729aa61b01c48c647d5113127a3b5b01e10381c44acf14c0 \
-                    size    243477948 \
-                    libreoffice-dictionaries-7.1.1.1.tar.xz \
-                    rmd160  fdd698927c93be94785ec49f550637f7e9bfe6fc \
-                    sha256  a456bb5feaa7d3eecb118670c976362d2e96170445686a67c897a2942c5d03a5 \
-                    size    47236084 \
-                    libreoffice-translations-7.1.1.1.tar.xz \
-                    rmd160  24aced0a554c6d2ef945b52f0bfd645e5a48b170 \
-                    sha256  fecb4ee40b3246f696a3a8dc476742b66f6f315d2e6eca539fc51a38f5586295 \
-                    size    184552736 \
+checksums           libreoffice-7.1.1.2.tar.xz \
```

Comment:
I would recommend using `${version}` in the filenames instead of hard-coding them; you can do that next update though if you like.

Reply:
Fixed.